package.json

```diff
@@ -43,11 +43,14 @@
     "memory-fs": "^0.3.0",
     "mocha": "^3.1.0",
     "nyc": "^8.3.1",
-    "xo": "^0.16.0"
+    "xo": "^0.16.0",
+    "webpack": "^1.13.2"
   },
   "scripts": {
     "pretest": "xo",
-    "test": "nyc mocha",
+    "test:webpack1": "npm i webpack@1 && nyc mocha",
+    "test:webpack2": "npm i webpack@beta && nyc mocha && npm i webpack@1",
```

Comment:
If you don't mind trying these out with `yarn` I think that would make it a bit faster and so on.